src/IO/AssimpLoader/AssimpAnimationDataLoader.cpp

```diff
@@ -1,29 +1,28 @@
-#include <Engine/Assets/AssimpAnimationDataLoader.hpp>
+#include <IO/AssimpLoader/AssimpAnimationDataLoader.hpp>
 
 #include <set>
 #include <assimp/scene.h>
 #include <assimp/mesh.h>
 
+#include <Core/File/AnimationData.hpp>
+#include <Core/File/KeyFrame/KeyRotation.hpp>
+#include <Core/File/KeyFrame/KeyScaling.hpp>
+#include <Core/File/KeyFrame/KeyTranslation.hpp>
 #include <Core/Log/Log.hpp>
-#include <Engine/Assets/AssimpWrapper.hpp>
-#include <Engine/Assets/AnimationData.hpp>
 
-#include <Engine/Assets/KeyFrame/KeyTranslation.hpp>
-#include <Engine/Assets/KeyFrame/KeyRotation.hpp>
-#include <Engine/Assets/KeyFrame/KeyScaling.hpp>
+#include <IO/AssimpLoader/AssimpWrapper.hpp>
 
 namespace Ra {
-namespace Asset {
-
+namespace IO {
 
 /// CONSTRUCTOR
-AssimpAnimationDataLoader::AssimpAnimationDataLoader( const bool VERBOSE_MODE ) : DataLoader< AnimationData >( VERBOSE_MODE ) { }
+AssimpAnimationDataLoader::AssimpAnimationDataLoader( const bool VERBOSE_MODE ) : DataLoader< Asset::AnimationData >( VERBOSE_MODE ) { }
```

Comment:
Seems like IO is for file IO stuff, and Asset is for storing data from IO stuff ...